src/native/libs/System.Security.Cryptography.Native/pal_evp_cipher.c

```diff
@@ -30,6 +30,9 @@ CryptoNative_EvpCipherCreate2(const EVP_CIPHER* type, uint8_t* key, int32_t keyL
         return NULL;
     }
 
+    // Required for OpenSSL 1.1 AES-KWP, no-op in OpenSSL 3.
+    EVP_CIPHER_CTX_set_flags(ctx, EVP_CIPHER_CTX_FLAG_WRAP_ALLOW);
```

Comment:
Without this flag the error is

> Interop+Crypto+OpenSslCryptographicException : error:0607B0AA:digital envelope routines:EVP_CipherInit_ex:wrap mode not allowed

This history of this flag is interesting. Originally, OpenSSL 1.1 started requiring this flag so that callers were sure their buffers were allocated correctly. AES-KWP doesn't fit neatly in to the "How much data am I going to write" allocation helpers, so this flag was introduced to help mitigate buffer overruns and callers had to explicitly indicate "Yes I know what I am doing and yes the buffers are correctly sized".